src/Common/src/System/NonSerializedAttribute.cs

```diff
@@ -5,10 +5,10 @@
 namespace System
 {
     [AttributeUsage(AttributeTargets.Field, Inherited = false)]
-    public sealed class NonSerializedAttribute : Attribute
```

Comment:
>public [](start = 4, length = 6)

Is this not a breaking change? or are we trying perhaps to mimic what we have on Desktop?